.gitignore

```diff
@@ -4,6 +4,8 @@ Pipfile.lock
 # node
 node_modules
 
+package-lock.json
```

Comment:
why is this being ignored?

Reply:
[a46f6ea](https://github.com/AMP-SCZ/dpdash/pull/72/commits/a46f6ea9b3065120f064f67fea39c6e7699261aa)

Reply:
[a1982cb](https://github.com/AMP-SCZ/dpdash/pull/72/commits/a1982cb4a32c868223e48cd0f5a98a72bdc2af45)